libs/common-services/src/main/java/uk/gov/di/ipv/core/library/config/ConfigurationVariable.java

```diff
@@ -32,7 +39,8 @@ public enum ConfigurationVariable {
     PUBLIC_KEY_MATERIAL_FOR_CORE_TO_VERIFY("clients/%s/publicKeyMaterialForCoreToVerify"),
     RETURN_CODES_ALWAYS_REQUIRED("self/returnCodes/alwaysRequired"),
     RETURN_CODES_NON_CI_BREACHING_P0("self/returnCodes/nonCiBreachingP0"),
-    SESSION_CREDENTIALS_TTL("self/sessionCredentialTtl");
+    SESSION_CREDENTIALS_TTL("self/sessionCredentialTtl"),
```

Comment:
Very minor thing - the enum is called SESSION_CREDENTIALS_TTL - plural. The path is singular.